descriptors/ModuleDescriptor-template.json

```diff
@@ -49,7 +49,10 @@
         {
           "methods": ["GET"],
           "pathPattern": "/invoice/invoice-lines",
-          "permissionsRequired": ["invoice.invoice-lines.collection.get"]
+          "permissionsRequired": ["invoice.invoice-lines.collection.get"],
```

Comment:
looks like the invoice-lines  interface dependency is missing, can you please add it

Reply:
dependency added.